microsoft/testsuites/ltp/ltp.py

```diff
@@ -87,6 +88,9 @@ def run_test(
         block_device: Optional[str] = None,
         temp_dir: str = "/tmp/",
         ltp_run_timeout: int = 12000,
+        # run full test if True, else run lite test
+        run_full_test: bool = False,
```

Comment:
If the list is not given, it can run full. So it doesn't need the additional parameter.